src/java/org/apache/sqoop/mapreduce/mainframe/MainframeConfiguration.java

```diff
@@ -33,4 +33,13 @@
   public static final String MAINFRAME_INPUT_DATASET_TAPE = "mainframe.input.dataset.tape";
 
   public static final String MAINFRAME_FTP_FILE_ENTRY_PARSER_CLASSNAME = "org.apache.sqoop.mapreduce.mainframe.MainframeFTPFileEntryParser";
+
+  public static final String MAINFRAME_FTP_TRANSFER_MODE = "mainframe.ftp.transfermode";
+
+  public static final String MAINFRAME_FTP_TRANSFER_MODE_ASCII = "ascii";
+
+  public static final String MAINFRAME_FTP_TRANSFER_MODE_BINARY = "binary";
+
+  // this is the buffer size used when doing binary ftp transfers from mainframe
+  public static final Integer MAINFRAME_FTP_TRANSFER_BINARY_BUFFER = 32760;
```

Comment:
Is this value always a good choice? I would make it configurable (if it isn't already), and use this value as default if said configuration is not present, so the user has a choice here.